integration/connection.tf

```diff
@@ -16,4 +16,40 @@ resource "materialize_connection" "schema_registry" {
   name                          = "schema_registry_connection"
   connection_type               = "CONFLUENT SCHEMA REGISTRY"
   confluent_schema_registry_url = "http://schema-registry:8081"
-}
\ No newline at end of file
+}
+
+resource "materialize_connection" "example_ssh_connection" {
+  name            = "ssh_example_connection"
+  schema_name     = "public"
+  connection_type = "SSH TUNNEL"
+  ssh_host        = "ssh_host"
+  ssh_user        = "ssh_user"
+  ssh_port        = 22
+}
+
+resource "materialize_connection" "kafka_conn_multiple_brokers" {
+  name            = "kafka_conn_multiple_brokers"
+  connection_type = "KAFKA"
+  kafka_brokers = [
+    {
+      broker = "kafka:9092"
+    },
+    {
+      broker = "kafka2:9092"
+    }
+  ]
+  kafka_sasl_username   = "sasl_user"
+  kafka_sasl_password   = format("%s.%s.%s", materialize_database.database.name, materialize_schema.schema.name, materialize_secret.kafka_password.name)
```

Comment:
Do you think we should add a fully qualified output for resources? Separate from this PR

Reply:
Good point!